src/gausskernel/storage/k2/config.cpp

```diff
@@ -25,12 +25,12 @@ namespace k2fdw {
 Config::Config() {
     const char* configFileName = getenv("K2_CONFIG_FILE");
     if (NULL == configFileName) {
-        K2LOG_W(log::k2fdw, "No config file given for K2 configuration in the K2_CONFIG_FILE env variable");
+        // K2LOG_W(log::k2fdw, "No config file given for K2 configuration in the K2_CONFIG_FILE env variable");
```

Comment:
I'd rather keep these here. Let's see if this works if we can make the Config variable in session.h as a thread_local

Reply:
That also seems to work. Updated the PR